src/mqttcm_connect.c

```diff
@@ -1738,12 +1718,12 @@ int mqtt_subscribe(char *comp, char *topic)
 		if(strcmp (comp, SUBSCRIBE_WEBCONFIG) == 0)
 		{
 			int subscribeId;
-
-			//Subscribe to wildcard topic "#"
-			stripAndAddModuleName(topic, SUBSCRIBE_WEBCONFIG, "#");
-			MqttCMInfo("Subscribing to wildcard topic - %s\n", topic);
-
-			rc = mosquitto_subscribe(mosq, &subscribeId, topic, 1);
+			if( Get_Mqtt_ClientId() != NULL)
+			{
+				snprintf(temp_topic, sizeof(temp_topic), "%s%s/#", MQTT_SUBSCRIBE_TOPIC, Get_Mqtt_ClientId());
```

Comment:
Instead of doing multiple get calls, assign to a variable and use with null check similar to 923, 924